.github/workflows/continuous-integration.yml

```diff
@@ -71,33 +71,6 @@ jobs:
         run: yarn --immutable
       - name: Typecheck And Build Code
         run: yarn typecheck && yarn build
-      - name: Upload bundle to artifacts
-        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # tag=v3
-        with:
-          name: bundle
-          path: dist/
-          if-no-files-found: error
-
-  NoModulesTesting:
```

Comment:
Why is this reverter

Reply:
If we don't bundle the deps `lodash` and `fast-deep-equal` then this test won't work